spec/models/spree/adjustment_spec.rb

```diff
@@ -491,5 +490,21 @@ module Spree
         end
       end
     end
+
+    context "return authorization adjustments" do
+      let!(:return_authorization) { create(:return_authorization, amount: 123) }
+      let(:order) { return_authorization.order }
+      let!(:return_adjustment) {
+        create(:adjustment, originator: return_authorization, order: order,
+                            adjustable: order, amount: 456)
+      }
+
+      describe "updating the adjustment" do
```

Comment:
I pretty much prefer to make it obvious that this is a unit test by rewriting it to `describe "#update!" do`. I feel like having this vague naming leads to unit tests not considering all input/output combinations. It feels to me like "we'll test some of the scenarios".

Also, when refactoring, I don't really know for sure which tests should I update to reflect a new design, whereas something like `#update!` makes it immediately obvious and it's git-greppeable.